tide/wall/MovieUpdater.cpp

```diff
@@ -46,6 +46,36 @@
 #include "network/WallToWallChannel.h"
 #include "scene/MovieContent.h"
 
+namespace
+{
+void _splitSideBySide(const FFMPEGPicture& image, PicturePtr& left,
+                      PicturePtr& right)
+{
+    const auto width = image.getWidth() / 2;
+    const auto height = image.getHeight();
+    const auto format = image.getFormat();
+
+    left = std::make_shared<FFMPEGPicture>(width, height, format);
+    right = std::make_shared<FFMPEGPicture>(width, height, format);
+
+    const uint numTextures = (format == TextureFormat::rgba) ? 1 : 3;
+    for (uint texture = 0; texture < numTextures; ++texture)
+    {
+        const auto uvSize = image.getTextureSize(texture);
+        const size_t lineWidth = uvSize.width();
+        const size_t targetWidth = lineWidth / 2;
+        for (size_t y = 0; y < size_t(uvSize.height()); ++y)
```

Comment:
this is not an issue performance wise now I guess? Thinking of vectorization possibilities maybe if we can't get 60 fps here :)